lab_3_keywords_textrank/main.py

```diff
@@ -152,7 +171,12 @@ def decode(self, encoded_tokens: tuple[int, ...]) -> Optional[tuple[str, ...]]:
                 sequence of string tokens
         In case of out-of-dictionary input data, None is returned
         """
-        pass
+        if not all(id in self._id2word for id in encoded_tokens):
+            return None
+        # for id in encoded_tokens:
+        #     if id not in self._id2word:
+        #         return None
+        return tuple(self._id2word.get(token) for token in encoded_tokens)
```

Comment:
you've already checked that all tokens are present in the `_id2word`, why do you still need to use `get`?